### ISSUE TITLE
数据集中的公式部分，有部分定位不准确，我觉得这部分是脏数据，有没有什么方法可以过滤掉这部分

### ISSUE THREAD

**Issue by VikingKang** [doc-analysis/DocBank#17]
![image](https://user-images.githubusercontent.com/23057462/96069917-c3270480-0ed1-11eb-8454-59177ac0ded3.png)
![image](https://user-images.githubusercontent.com/23057462/96069950-d043f380-0ed1-11eb-9cb9-7f21d7479d4e.png)
两张图片是在sample某一张图片的ann.jpg和ari.jpg对应的部分截图

**Comment by liminghao1630** [comment]
Hi, this kind of coordinate shift naturally occurs in PDFMiner and hardly to be avoided.

**Comment by VikingKang** [comment]
Is there a way to filter by font or cid symbol？

**Comment by ltss1988** [comment]
Have you solved the problem？

**Comment by Bonjour123** [comment]
Up ! Am I right if I say that this render the full dataset completely useless ? Or does this affect only a small part of it ?

**Comment by liminghao1630** [comment]
@VikingKang @ltss1988 @Bonjour123 
This problem affects only the equation, and hard to filter. These tokens seem to cover other tokens in the rendering, but they will not affect the actual use.

**Comment by Bonjour123** [comment]
Well, if so can you help me understand why I need to apply some random horizontal dilatation to make the bounding boxes match ? 
Here are some randomly picked examples:
- 1.tar_1401.0001.gz_infoingames_without_metric_arxiv_0
- 1.tar_1401.0001.gz_infoingames_without_metric_arxiv_24
- 1.tar_1401.0007.gz_hhmerge_3

I first thought of some bugs in the visualisation implementation, but I get the same problem using both PIL (red) and cv2 (blue). 

<img src="https://user-images.githubusercontent.com/17679888/102592842-5bc94300-4114-11eb-8e3a-2e9e222ca2ca.png" width="400" >
<img src="https://user-images.githubusercontent.com/17679888/102592846-5ec43380-4114-11eb-9607-d55339955e5a.png" width="400" >
<img src="https://user-images.githubusercontent.com/17679888/102592850-5f5cca00-4114-11eb-884a-d8d02a3c7ca5.png" width="400" >
<img src="https://user-images.githubusercontent.com/17679888/102592852-5ff56080-4114-11eb-9976-ca8b3531ae6d.png" width="400" >

**Comment by Bonjour123** [comment]
@liminghao1630 @VikingKang @ltss1988  Don't you have the same problem ?

**Comment by liminghao1630** [comment]
#19 @Bonjour123

**Comment by Permafacture** [comment]
Yes I'm also seeing that the unusually tall CID characters have bounding boxes that are shifted up. (Not the issue Bonjour is talking about). I'm surprised that such an obvious, systematic, and pervasive issue (affects ~1/3 of samples) wasn't caught and fixed at creation time, especially since the PDFs weren't released and so there's no way for anyone to fix this issue at the root cause after the fact. I'm also not finding an ad hoc work around posted anywhere.
